api/providers/views.py

```diff
@@ -688,13 +689,20 @@ def get_default_queryset(self):
         default_schema_id = provider.default_schema.id if provider.default_schema else None
         if not default_schema_id:
             return provider.schemas.get_latest_versions(request=self.request, invisible=True).filter(active=True)
-        schemas = provider.schemas.values('name').annotate(version=Max('schema_version'))
-        annotated_schemas = schemas.annotate(default_schema_ordering=Case(
+        latest_versions = RegistrationSchema.objects.values('name').annotate(latest_version=Max('schema_version'))
+        annotated = RegistrationSchema.objects.all().annotate(
+            latest_version=Subquery(
+                latest_versions.filter(name=OuterRef('name')).values('latest_version')[:1],
+                output_field=IntegerField(),
+            ),
+        )
+        filtered = annotated.filter(schema_version=F('latest_version'), providers=provider)
+        filtered = filtered.annotate(default_schema_ordering=Case(
             When(id=default_schema_id, then=Value(1)),
             default=Value(0),
-            output_field=PositiveIntegerField(),
+            output_field=IntegerField(),
```

Comment:
probably doesn't really matter if this is `PositiveIntegerField` or `IntegerField` :)

Reply:
Probably not. 🤷